ui/components/ConditionsTable.tsx

```diff
@@ -18,31 +10,23 @@ type Props = {
 
 function ConditionsTable({ className, conditions }: Props) {
   return (
-    <div className={className}>
-      <TableContainer>
-        <Table>
-          <TableHead>
-            <TableRow>
-              <TableCell>Type</TableCell>
-              <TableCell>Status</TableCell>
-              <TableCell>Reason</TableCell>
-              <TableCell>Message</TableCell>
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {_.map(conditions, (c) => (
-              <TableRow key={c.timestamp}>
-                <TableCell>{c.type}</TableCell>
-                <TableCell>{c.status}</TableCell>
-                <TableCell>{c.reason}</TableCell>
-                <TableCell>{c.message}</TableCell>
-              </TableRow>
-            ))}
-          </TableBody>
-        </Table>
-      </TableContainer>
-    </div>
+    <DataTable
+      fields={[
+        { label: "Type", value: (c) => c.type },
```

Comment:
If I remember correctly how the API for this work, this should simplify things:
```suggestion
        { label: "Type", value: 'type' },
```

Reply:
YES

Reply:
> Wait - isn't it getting the type from the Condition object?

Yes. If the table finds a function, it runs it; else it returns the value. This allows you to get the value for a column from somewhere else on the object if you need to, but also makes the default case easy.

Reply:
> 

Wow COOL. fixed.